website/docs/index.html.markdown

```diff
@@ -101,15 +101,15 @@ $ terraform plan
 
 The following arguments are supported in the `provider` block:
 
-* `bluemix_api_key` - (optional) The Bluemix API key. You must either add it as a credential in the provider block or source it from the `BM_API_KEY` (higher precedence) or `BLUEMIX_API_KEY` environment variable. The key is required to provision Cloud Foundry or IBM Container Service resources, such as any resource that begins with `ibm` or `ibm_container`.
```

Comment:
In the provider block this is an optional parameter. Let us keep it that way.

Reply:
These changes were requests from @davetropeano

Reply:
Sure the reason for this is explained in the below review comment.